sql/database.sql

```diff
@@ -3384,6 +3384,7 @@ CREATE TABLE `layout_group_properties` (
   grp_save_close  tinyint(1)     not null default 0,
   grp_init_open   tinyint(1)     not null default 0,
   grp_referrals   tinyint(1)     not null default 0,
+  grp_unchecked   tinyint(1)     not null default 0,
```

Comment:
Hey @bradymiller it's right here :wink: 

Or did you mean another addition?

Reply:
nope, this is good. I just need some more coffee :)